src/app/collection/pages/nice-challenge/components/philosophy/philosophy.component.scss

```diff
@@ -0,0 +1,121 @@
+@import '~_vars.scss';
+@use '~collection-globals.scss';
+
+$nice-blue: #3693CC;
+
+.wrapper {
+  display: flex;
+  justify-content: space-between;
+
+  .philosophy-content {
+    display: flex;
+    flex-basis: 75%;
+    justify-content: center;
+
+    .grid {
+      width: 100%;
+      display: grid;
+      grid-template-columns: 50% 50%;
+      grid-row: auto auto;
+      justify-content: space-evenly;
+      grid-row-gap: 20px;
+
+      .grid-item {
+        padding: 20px;
```

Comment:
Personally, I think there is just a little too much padding, but if you and others like it, it all looks good!